src/gordon_gcp/clients/gdns.py

```diff
@@ -153,11 +153,12 @@ def get_managed_zone(self, zone):
 
         # to limit the amount of data across the wire; also makes it
         # easier to create GCPResourceRecordSet instances
-        fields = ('rrsets/name,rrsets/kind,rrsets/rrdatas,'
-                  'rrsets/type,rrsets/ttl,nextPageToken')
-        params = {
-            'fields': fields,
-        }
+        if not params:
```

Comment:
Ah,  good point.  I'll bump it down into the if block where fields are added and nix the bit about data, because it seems like it only removes one field, and that seems less relevant